tensorboard-controller/rockcraft.yaml

```diff
@@ -0,0 +1,60 @@
+name: tensorboard-controller
+summary: An image for Tensorboard controller
+description: |
+  This image is used as part of the Charmed Kubeflow product. The controller allows users to
+  create a custom resource "TensorBoard"
+version: v1.7.0_22.04_1 # version format: <KF-upstream-version>_<base-version>_<Charmed-KF-version>
+license: Apache-2.0
+base: ubuntu:22.04
+services:
+  tensorboard:
+    override: replace
+    summary: "tensorboard-controller service"
+    startup: enabled
+    user: ubuntu
+    command: "/manager"
+platforms:
+  amd64:
+
+parts:
+  tensorboard-controller:
+    plugin: go
+    source: https://github.com/kubeflow/kubeflow
+    source-type: git
+    source-tag: v1.7-branch # upstream branch
+    build-snaps:
+      - go
+    build-packages:
+      - apt
+      - bash
+    build-environment:
+      - CGO_ENABLED: 0
+      - GOOS: linux
+    stage-packages:
+      - bash
+    override-build: |
+      set -xe
+      cd $CRAFT_PART_SRC/components/tensorboard-controller/
+
+      # build
+      go mod tidy
+      go mod download all
+      go build -a -o manager main.go
+
+      # install build artifacts
+      mkdir -p $CRAFT_PART_INSTALL/
+      cp manager $CRAFT_PART_INSTALL/
+
+      # security requirement
```

Comment:
can you put this into another part?

Reply:
ACK I will add a `security-team-requirement` part